src/decoder/mod.rs

```diff
@@ -186,13 +186,16 @@ fn decode_array<R: Read + ?Sized>(reader: &mut R, utf8_lossy: bool) -> DecoderRe
 }
 
 fn decode_bson<R: Read + ?Sized>(reader: &mut R, tag: u8, utf8_lossy: bool) -> DecoderResult<Bson> {
-    use spec::ElementType::*;
```

Comment:
The import of `ElementType::JavaScriptCodeWithScope` conflicted with the struct, so I changed this to only import `ElementType` itself.